continuous_integration/travis/install.sh

```diff
@@ -18,6 +18,7 @@ download_code()
     if [ "$PR" != "false" ] ; then
         cd $FULL_SPYDER_CLONE
         git fetch origin pull/$PR/head:travis_pr_$PR
+        git checkout travis_pr_$TRAVIS_PULL_REQUEST
```

Comment:
This is done in `build_test.sh`, so you don't need to do it here:

https://github.com/spyder-ide/spyder/blob/master/continuous_integration/travis/build_test.sh#L10